tests/test_helpers.py

```diff
@@ -1,17 +1,37 @@
-from open_mastr.utils.constants import API_LOCATION_TYPES
+import pytest
+import os
+from os.path import expanduser
+import sys
+import random
+from os.path import join
+from datetime import datetime
+import pandas as pd
+from open_mastr import Mastr
+
+from open_mastr.utils import orm
+from open_mastr.utils.constants import API_LOCATION_TYPES, TECHNOLOGIES, API_DATA_TYPES, ADDITIONAL_TABLES
+from open_mastr.utils.config import get_data_version_dir, create_data_dir
 from open_mastr.utils.helpers import (
     validate_parameter_format_for_download_method,
     validate_parameter_format_for_mastr_init,
     validate_api_credentials,
     transform_data_parameter,
     data_to_include_tables,
+    session_scope,
+    create_db_query,
+    db_query_to_csv,
+    reverse_unit_type_map
 )
-import pytest
-import sys
-from datetime import datetime
-from open_mastr import Mastr
 
 
+# Check if db is empty
+_db_exists = False
+_db_path = os.path.join(expanduser("~"), ".open-MaStR", "data", "sqlite", "open-mastr.db") # FIXME: use path in tmpdir when implemented
+if os.path.isdir(_db_path):
```

Comment:
Either if  should be replaced with `os.path.isfile(_db_path)` or below in the for loop the entry must be checked and the path should be the sqlite directory. 

Currently,` _db_exists` is never going to be True, right?

Reply:
I renamed one parameter not to overwrite _db_path. If it seems correct for you, you can resolve conversation.